### PR TITLE
Unauthenticated user can access chartering page

### PR DESCRIPTION
**Pull request by SnowyNate**
Fixed issue of authenticated user being able to access the chartering page, and added functionality to redirect a logged out user to the landing page if they attempt to access the chartering page while logged out.

### CODE REVIEW

File: src/pages/Create.tsx [hunk 1]
```diff
@@ -20,6 +20,10 @@ import OrgRequirements from "../utils/OrgRequirements";
 
 import { PUBLIC_URL } from "../constants";
```
Comment (Rahuldeb5) [inline src/pages/Create.tsx:21]: Add import { useSnackbar } from "notistack";

File: src/pages/Create.tsx [hunk 2]
```diff
@@ -67,7 +71,9 @@ const multilineStyle: CSSProperties = {
 };
 
 const Create = () => {
-    const user = useContext(UserContext);
+    const navigate = useNavigate();
+    const user : UserContextType = useContext(UserContext);
+
     const { enqueueSnackbar } = useSnackbar();
```
Comment (Rahuldeb5) [inline src/pages/Create.tsx:77]: It seems like you're not using the snackbar.

**Review by Rahuldeb5 — COMMENTED** [review]
Your code seems to work correctly and the logic is fine. I think it would be a tad bit nicer if you added a snackbar or alert dialog to notify the unauthenticated user to sign in before they can access that page.

**Comment by SnowyNate** [comment]
Got it. Will try to add that now.

File: src/index.tsx [hunk 3]
```diff
@@ -8,9 +8,7 @@ const root = ReactDOM.createRoot(
     document.getElementById("root") as HTMLElement,
 );
 root.render(
-    <React.StrictMode>
```
Comment (TheEgghead27) [inline src/index.tsx:11]: Remind me why you are making this change?

Also, don't forget to `npm run prettier`

**Review by TheEgghead27 — COMMENTED** [review]
Great work here, I am currently at work so I cannot test the changes, and I would like to know why we are removing the strict tags in this PR

**Comment by SnowyNate** [comment]
Thank you, and no worries. When I originally ran the code, the message (as well as a console.log that I ran) was appearing twice. After doing some research, this was because of react's strict mode, which was, in fact, the cause. Apparently this only affects the development server.

File: src/index.tsx (hunk 3 above)
Comment (SnowyNate) [reply src/index.tsx:11]: Will run when I get back home.

**Review by Rahuldeb5 — APPROVED** [review]

**Pull request merged**